### PR TITLE
Feature/delete ingredient

### PR DESCRIPTION
**Pull request by stepris**
Please have a look :)

### CODE REVIEW

File: components/RecipeForm/IngredientInput.tsx [hunk 1]
```diff
@@ -6,10 +6,13 @@ import {
 } from '@/components/RecipeForm/recipeStyles';
 import { IngredientInputProps } from '@/types/IngridientInput.types';
 import { HandleChangeParams } from '@/types/RecipeForm.types';
+import DeleteIcon from '@/public/icons/buttons/ic_fluent_delete_24_regular.svg';
+import { StyledDeleteIconWrapper } from '@/components/RecipeForm/recipeStyles';
```
Comment (stekug) [inline components/RecipeForm/IngredientInput.tsx]: It would be better to call this "StyledDeleteIngredientButton" because - its a button :)

**Review by stekug — APPROVED** [review]
When we fix the button name, we can merge :)
Good Job! Really well done!

File: components/RecipeForm/IngredientInput.tsx (hunk 1 above)
Comment (stepris) [reply components/RecipeForm/IngredientInput.tsx]: You're right, thanks! :)

**Comment by stepris** [comment]
Thank you, I changed the naming! 💪

**Review by stekug — APPROVED** [review]
Perfect!

**Pull request merged**